src/main/app/forms/validation/validationErrors.ts

```diff
@@ -1,6 +1,8 @@
 export class ValidationErrors {
   static readonly TEXT_TOO_LONG: string = 'You’ve entered too many characters'
   static readonly AMOUNT_TOO_HIGH: string = 'Are you sure this is a valid value?'
+  // TODO Get Ranjit to check that AMOUNT_TOO_HIGH has the correct content
```

Comment:
@jacmurphy ?

Reply:
Ive addressed this with @rattur.
Once this story is merged into master, his story needs to be refactored. There doesn't seem to be a record of the correct content so it will need to be clarified with UX/Content team to ensure it doesn't cause issues